django_weasyprint/views.py

```diff
@@ -90,7 +92,7 @@ def rendered_content(self):
         Returns rendered PDF pages.
         """
         document = self.get_document()
-        return document.write_pdf()
+        return document.write_pdf(options=self._options)
```

Comment:
Same as above.

Reply:
yes, should use **; no need for copy/setdefault, i think, since we do not provide any other kwargs here (valid kwargs other than `options` here are: [`target`, `zoom`, `finisher`](https://doc.courtbouillon.org/weasyprint/stable/api_reference.html#weasyprint.document.Document.write_pdf)).